.github/ISSUE_TEMPLATE/story.md

```diff
@@ -8,32 +8,25 @@ assignees: ''
 ---
 
 # Story
+As a _, so that _, I need _.
 
-## Sentence
+### Pre-conditions
```

Comment:
We should only decrease by 1 heading size.  So, instead of `###`, this should be `##`.  This is an accessibility thing.

Reply:
Thanks for calling that out @halprin -- fixed. (I didn't care for the extraneous dividing lines GH was adding, but I didn't know for certain there were accessibility implications.)